### ISSUE TITLE
Saccharomyces TIM10, TIM12 gene function annotations

### ISSUE THREAD

**Issue by LiNiMGI** [geneontology/go-annotation#5581]
Hi Stacia @srengel 
I am working on a project of “human genes no IBA MF’ to see whether there is data that can be used to PAINT the gene tree.
And I noticed:
TIM10, yeast annotated to protein transporter activity IDA/IMP, 
TIM12 yeast annotated to protein transmembrane transporter activity IDA/IMP
While human annotated the ortholog TIMM10 to membrane insertase activity IDA

Do you think ‘membrane insertase activity GO:0032977” better describes the yeast TIM10 and TIM12 function here?
Thanks,
Li

**Comment by ValWood** [comment]
I think tim8-tim10 (and hence tim12) might be a chaperone rather than an insertase (although it supports the insertion)
and tim22 complex is the insertase.

In that case the complex term
GO:0042719 mitochondrial intermembrane space protein transporter complex
seems wrong
 
@hattrill ?

**Comment by hattrill** [comment]
I think this one is a bit tricky: the definition GO:0042719 mitochondrial intermembrane space protein transporter complex is:
Soluble complex of the mitochondrial intermembrane space composed of various combinations of small Tim proteins; acts as a protein transporter to guide proteins to the Tim22 complex for insertion into the mitochondrial inner membrane.
So that seems ok

In these protein transporters, chaperones are involved in the process of transport - they kind of pull the subunits through the transport machinery and they are not protein folding chaperones. contribute_to ;-) ?

**Comment by hattrill** [comment]
Or may be just 'protein carrier chaperone'?

**Comment by hattrill** [comment]
and Timm22 is the insertase?

**Comment by ValWood** [comment]
OK that makes more sense. That's  what I  thought originally!

However:

 TIM22 mitochondrial import inner membrane insertion complex (GO:0042721)
A multi-subunit complex embedded in the mitochondrial inner membrane that mediates the inner membrane insertion of multi-transmembrane spanning proteins that contain internal targeting elements. In yeast cells, TIM22 is a 300-kDa complex, consisting of four membrane integral subunits, Tim22, Tim54, Tim18 and Sdh3, and a peripheral chaperone complex consisting of the small TIM proteins, Tim9-Tim10-Tim12. [PMID:27554484, PMID:12191765]

So
Tim22  membrane insertase activity (GO:0032977) (contributes_to)
Tim54  membrane insertase activity (GO:0032977) (contributes_to)
Tim18 GO:0043495 protein-membrane adaptor activity
Sdh23 GO:0043495 protein-membrane adaptor activity (note this is also adaptor for Succinate DH complex)

Tim9 protein carrier chaperone (GO:0140597)
Tim10 protein carrier chaperone (GO:0140597)

?

The one thing I find odd about this is that 
protein carrier chaperone is a parent of membrane insertase activity, but these are different activities molecularly

**Comment by hattrill** [comment]
Yes, I can kind of see why insertases are a classes as protein carrier chaperone - mechanisically, they are quite strange - they mediate the partition of the hydrophobic portion of the TM protein into the bilayer by providing hydrophobic interaction interfaces. However, it becomes blurred with TM transport as a TIM23 variant TIM23SORT acts as an insertase rather than a TM transporter...

For Tim9 and Tim10, GO:0140318 protein transporter activity is just as valid as protein carrier chaperone when comparing the MF defs. As they are intergral to transport, perhaps protein transporter activity  is ok (just not TM transporter activity)

**Comment by ValWood** [comment]
Most publications depict and describe tim10/12 as actual chaperones:

https://www.sciencedirect.com/science/article/pii/S002228361200784X
<img width="782" alt="Screenshot 2024-12-12 at 10 52 04" src="https://github.com/user-attachments/assets/6d2a9053-3e16-487c-83fa-5262fe93c73c" />

They are in the intermembrane space  and bind to the cargo. But I don't think it is correct to describe as transporters  (even though they are involved in the transport process), because the movement is 'not directed' which is necessary to describe a a transporter.

That's why  "protein carrier chaperone" isn't a transporter.

The thing I find odd is  describing an integrate as a "protein carrier chaperone", but maybe that is OK?

**Comment by hattrill** [comment]
I also find the integrase odd as a chaperone - I think with both integrases and the tim9-10 there are arguments to have them as transporters and/or chaperones.

**Comment by srengel** [comment]
hi guys, what is the conclusion here?  please advise.

**Comment by hattrill** [comment]
@srengel I would say, do nothing for now as the instructions for using  GO:0140318 protein transporter activity suggest that this term was made with this complex in mind, so we will need to make sure that we are clear in the difference between a protein transporter and a chaperone - this might mean that we redefine the former to only act on folded proteins and the latter on unfolded, for example. I am going to give this some thought as it is not the only example of a chaperone that is integrated into TM protein transport.

**Comment by ValWood** [comment]
@LiNiMGI  sounds like a great project. Hopefully, they won't all be a can of worms!

**Comment by pgaudet** [comment]
Thanks @hattrill 
GO:0140318 was created in 2019 (#17073), when we were less clear about chaperones and transporters and carriers. 

Would you please suggest improvements to more cleary describe the function and where the term should be in the ontology?

**Comment by hattrill** [comment]
My broad concern is that is no real way to tell between 'protein transporter activity' and 'protein carrier chaperone':

**protein transporter activity**
Directly binding to a specific protein and delivering it to a specific cellular location. PMID:18706423 
synonyms:
protein transport chaperone 	related
protein carrier activity 	related

**protein carrier chaperone**
Binding to and carrying a protein between two different cellular components by moving along with the target protein. PMID:7628437 
protein carrier activity 	exact
protein chaperone 	broad

I agree that there are 'chaperones' involved in TM transport, but it would be difficult choose the correct term and their MFs are not all equivalent. I would like a consistent way to group/describe these activities as they are very much coupled to TM transport/membrane protein insertion but I do not think we have the correct terms to describe them. I also wonder whether GO:0051082 unfolded protein binding is often used instead of something more satisfactory. I have divided the examples I have come across into three broad classes, but there may be more:

1. **Holdases**:
e.g. Tim8-Tim13 - soluble, mitochondrial intermembrane space chaperone. Tim8-Tim3 complex mediates the import of a small subset of proteins 
and Tim9-Tim10 Is described as a peripheral membrane protein of the TIM22 complex (GO:0042721) or soluble, mitochondrial intermembrane space chaperone
They act to bind unfolded proteins as they emerge from the TOMM complex and keep them in a translocation-competent conformation in the mitochondrial intermembrane space to present them to the SAM or TIM22 complex.

('holdase activity' is general chaperone activity https://doi.org/10.1016/bs.mie.2024.07.051 )


2. **ATPase rachets/motors**:
2a. mitochondrial HSP70/HSPA9: The majority of mtHSP70/HSPA9 acts as a protein folding chaperone in the mitochondrial matrix. A small proportion is part of the presequence translocase-associated motor (PAM) is a subcomplex of the TIM23 complex. In this, mtHSP70/HSPA9 binds precursor proteins to promote their unidirectional translocation across the two mitochondrial membranes and into the mitochondrial matrix by pulling them through the TIM23 complex pore. This is achieved by cycles of high-low affinity binding driven by ATP hydrolysis that generates a pulling force. 
2b. ER HSP70/BiP: Which assists nascent peptide transit through the Sec61 translocon into the ER. This is slightly different to the mechanism of HSP70/HSPA9, ER HSP70/BiP does not generate pulling forces but stabilizes nascent chains by preventing backsliding through ATP-driven cycles of binding and release. PMID:34360841

(note: looks like annotations to GO:0015450 protein-transporting ATPase activity could do with a review)

3. **Unfoldases**:
e.g. PEX1/PEX6 (Peroxisomal membrane extraction complex). An ATPase chaperone complex that interacts with monoubiquitinated Pex5 cargo receptor in the peroxisomal retrotranslocon complex. Mediates ATP-dependent unfolding of Pex5 and extracts it into the cytosol. https://www.nature.com/articles/s41467-017-02474-4

(note protein unfoldase activity is a broad synonym for GO:0140545 ATP-dependent protein disaggregase activity, but is this activity not also a type of chaperone?)

It is also unclear to me why some activities* are classed as transport and others as chaperone:

```
protein transporter activity GO:0140318
	|_membrane protein dislocase activity GO:0140567 (ATP-dep extraction)*
	|_protein transmembrane transporter activity GO:0008320
	|_lipoprotein releasing activity GO:0140306 (ATP-dependent extrusion)*

protein carrier chaperone GO:0140597
	|_lipoprotein carrier activity GO:0140598*
	|_histone chaperone activity GO:0140713
	|_membrane insertase activity GO:0032977*
```

Just also note this here, so it doesn't get lost:
GO:0061742 chaperone-mediated autophagy translocation complex is capable_of protein carrier chaperone - incorrect as this has been used to annotate LAMP2A.  LAMP2A is not a chaperone, chaperones work upstream of LAMP2A which forms a channel/translocon. So, should be capable_of protein transmembrane transport GO:0071806 PMC10549195

So, in summary, I think we need some new terms and a bit of a review of these 2 branches. I can suggest some more specific terms, but I would like to see if there is some broad agreement on this before that.

**Comment by hattrill** [comment]
Perhaps I was not clear, but those classes: holdases, etc are chaperone-type terms not transporter type terms.

**Comment by ValWood** [comment]
OK, in this case, do we have the answer?
tim10/tim12 are holdases, not transporters.
they bind to the unfolded protein and act as holdases as part of the MIA import pathway.

But we don't have a term for "holdase activity. Perhaps we should?  (we have only unfolded protein binding). Perhaps that would help?

**Comment by hattrill** [comment]
Hi @ValWood I do think that we need some more terms to describe the different types of chaperone activity (I am in favour of a holdase activity for tim10/tim12 type chaperones). Think it would be good if we could get together with @pgaudet in the new year and come up with solution that makes sense for all chaperones that are involved in transport systems (don't think it will be too hard, but it would be good to discuss this over zoom).

**Comment by ValWood** [comment]
Agreed!

**Comment by raymond91125** [comment]
> GO:0140309 unfolded protein carrier activity holdase-carrier chaperone | exact holdase | broad is the term to use for tim 9,10,8,13
> 
> Hopefully it will be renamed to holdase-carrier chaperone or even "holdase"

@ValWood Per this discussion https://github.com/geneontology/go-ontology/issues/30552#issuecomment-3487576073, I thought the plan was to create a more general "holdase" MF term. Is that not true anymore? Holdase apparently not involved in transport PMID:25681016 A first line of stress defense: small heat shock proteins and their function in protein homeostasis.

**Comment by raymond91125** [comment]
The definition of GO:0140309 unfolded protein carrier activity is "A protein carrier activity that binds to a protein in an unfolded state and escorts it between two different cellular components." When a holdase functions in a way that does not specify TWO different cellular components, is it still a carrier? 

A more general definition for a holdase MF could be: Binding to unfolded or partially folded proteins to prevent aggregation and maintain soluble, folding-competent state.

**Comment by edwong57** [comment]
done
